tests/containers/podman_ipv6.pm

```diff
@@ -35,14 +36,16 @@ sub run {
     my $podman = $self->containers_factory('podman');
     install_packages('bind-utils');
 
+    # Prepare variables
```

Comment:
IMHO comments needs to say something valuable , something which you can not guess from simply looking into the code below , comments like "Declare a function" , "sending message" , "prepare variables" - does not bring any valuable info hence just eating the space

Reply:
Done.